xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/AmmIT.java

```diff
@@ -288,6 +293,562 @@ void depositAndWithdraw() throws JsonRpcClientErrorException, JsonProcessingExce
+    KeyPair issuerKeyPair = createRandomAccountEd25519();
+    KeyPair traderKeyPair = createRandomAccountEd25519();
+    FeeResult feeResult = xrplClient.fee();
+
+    AccountInfoResult issuerAccount = scanForResult(
+        () -> this.getValidatedAccountInfo(issuerKeyPair.publicKey().deriveAddress())
+    );
+    AccountInfoResult traderAccount = scanForResult(
+        () -> this.getValidatedAccountInfo(traderKeyPair.publicKey().deriveAddress())
+    );
+
+    // enable rippling
+    enableFlag(issuerKeyPair, issuerAccount.accountData().sequence(), feeResult, AccountSetFlag.DEFAULT_RIPPLE);
+
+    // enable Allow Trustline Clawback
+    enableFlag(issuerKeyPair, issuerAccount.accountData().sequence().plus(UnsignedInteger.ONE),
+        feeResult, AccountSetFlag.ALLOW_TRUSTLINE_CLAWBACK);
+
+    // create Trustline from issuer to trader for currency TST and define two currencies
+    String testCurrency = "TST";
+    TrustSet trustSet = TrustSet.builder()
+        .account(traderKeyPair.publicKey().deriveAddress())
+        .fee(FeeUtils.computeNetworkFees(feeResult).recommendedFee())
+        .sequence(traderAccount.accountData().sequence())
+        .signingPublicKey(traderKeyPair.publicKey())
+        .limitAmount(IssuedCurrencyAmount.builder()
+            .currency(testCurrency)
+            .issuer(issuerKeyPair.publicKey().deriveAddress())
+            .value("1000")
+            .build()
+        )
+        .flags(TrustSetFlags.builder().tfClearNoRipple().build())
+        .build();
+
+    SingleSignedTransaction<TrustSet> signedTrustline =
+        signatureService.sign(traderKeyPair.privateKey(), trustSet);
+    SubmitResult<TrustSet> trustlineSubmitResult = xrplClient.submit(signedTrustline);
+    assertThat(trustlineSubmitResult.engineResult()).isEqualTo(TransactionResultCodes.TES_SUCCESS);
+
+    // create Trustline from issuer to trader for ZFT
+    TrustSet trustSet2 = TrustSet.builder()
+        .account(traderKeyPair.publicKey().deriveAddress())
+        .fee(FeeUtils.computeNetworkFees(feeResult).recommendedFee())
+        .sequence(traderAccount.accountData().sequence().plus(UnsignedInteger.ONE))
+        .signingPublicKey(traderKeyPair.publicKey())
+        .limitAmount(
+            IssuedCurrencyAmount.builder()
+                .currency(xrpl4jCoin)
+                .issuer(issuerKeyPair.publicKey().deriveAddress())
+                .value("1000000")
+                .build()
+        )
+        .flags(TrustSetFlags.builder().tfClearNoRipple().build())
+        .build();
+
+    SingleSignedTransaction<TrustSet> signedTrustline2 =
+        signatureService.sign(traderKeyPair.privateKey(), trustSet2);
+    SubmitResult<TrustSet> trustlineSubmitResult2 = xrplClient.submit(signedTrustline2);
+    assertThat(trustlineSubmitResult2.engineResult()).isEqualTo(TransactionResultCodes.TES_SUCCESS);
+
+    // send payment of ZFT to trader wallet from issuer
+    Payment paymentZFT = Payment.builder()
```

Comment:
This naming convention breaks our Checkstyle plugin. Need to replace all instances of `...ZFT` with `Zft`. For example:

```suggestion
    Payment paymentZft = Payment.builder()
```

Reply:
fixed